packages/mdc-notched-outline/mdc-notched-outline.scss

```diff
@@ -28,57 +28,58 @@
 
 // Notched Outline is intended for use by multiple components, but its styles should only be emitted once when bundled
 @include mdc-base-emit-once("mdc-notched-outline") {
-  .mdc-notched-outline,
-  .mdc-notched-outline__idle {
+  .mdc-notched-outline {
```

Comment:
It looks like this branch removes the idle outline element completely, and thus also removes the fade. This seems a bit jarring... we might want to run that by one of our motion designers.

I don't think keeping the idle element would present too much of a problem? (Since now the borders should behave exactly the same, since the notched outline is also simply an element of the same size using CSS borders, rather than an SVG.)

Reply:
I plan to apply the transition properties from the idle element to the notched outline to animate color. That should result in a very similar appearance.

Reply:
We'll have to see how that performs. Opacity can be GPU-composited; color can't.

Reply:
The previous idle element transitioned border-color and opacity.